t/024-access/request_body.t

```diff
@@ -170,3 +170,22 @@ Expect: 100-Continue
 http finalize request: 500, "/echo_body?" a:1, c:2
 http finalize request: 500, "/echo_body?" a:1, c:0
 --- log_level: debug
+
+
+
+=== TEST 9: test HTTP2 reading request body was disabled
+--- config
+    location /echo_body {
+        lua_need_request_body on;
+        access_by_lua '
+            ngx.print(ngx.var.request_body or "nil")
+        ';
+        content_by_lua 'ngx.exit(ngx.OK)';
+    }
+--- http2
+--- request eval
+"POST /echo_body
+hello\x00\x01\x02
+world\x03\x04\xff"
+--- response_body eval
+"nil"
```

Comment:
we also need to test the error log

Reply:
It can not contain any related error log, because http2 read body was skip in https://github.com/openresty/lua-nginx-module/pull/2174/files#diff-222c2a4762e6ad44f96e551ae24c481e1577c1c9791638e6bd017daaf36b8600R145